netimpair/__init__.py

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env python
```

Comment:
Hmm I wonder if I rewrote these commits with a new version of git now it would keep attribution (though not entirely sure it's not - just haven't looked at this in years).

Ideally all `git blame` lines still attribute the original author.

Reply:
Interesting. What happened to make it lose that attribution? Is it just reformatting?

Reply:
I'm actually not saying it **did**.
I'm wondering if it did. I know old git had troubles with file renaming and when I made these changes it was quite a few years ago ;)

I can check the commits if I can find my own fork :rofl: